site-src/v1alpha2/guides/multiple-ns.md

```diff
@@ -3,150 +3,130 @@
+By managing this certificate through the single shared Gateway they are able
 to centrally control security without directly involving application teams.
 
 The logical relationship between the Gateway API resources looks like this:
 
 ![Cross-Namespace routing](../images/cross-namespace-routing.svg)
 
-## Cross-namespace Route binding
+## Cross-namespace Route Attachement
 
-[Route binding](/concepts/api-overview/#route-binding) is an important concept 
-that dictates how Routes and Gateways select each other to apply routing
-configuration to a Gateway. It is especially relevant when there are multiple
-Gateways and multiple Namespaces in a cluster. Gateway and Route binding is
-bidirectional - a binding can only exist if the Gateway owner and Route owner
-owner both agree to the relationship. This bi-directional relationship exists
-for two reasons:
+[Route attachment][attaching] is an important concept that dictates how Routes
+and Gateways select each other to apply routing configuration to a Gateway. It
+is especially relevant when there are multiple Gateways and multiple Namespaces
+in a cluster. Gateway and Route attachment is bidirectional - attachment can
+only succeed if the Gateway owner and Route owner owner both agree to the
+relationship. This bi-directional relationship exists for two reasons:
 
-- Route owners don't want to overexpose their applications and don't want 
+- Route owners don't want to overexpose their applications and don't want
 their apps to be accessible through paths they are not aware of.
-- Gateway owners don't want apps using certain Gateways they should not be 
-using. An internal application shouldn't be exposed through a public Gateway 
+- Gateway owners don't want apps using certain Gateways they should not be
+using. An internal application shouldn't be exposed through a public Gateway
 for example.
 
-
 As a result, Gateways and Routes have independent control to determine which
-resources they permit binding with. It is a handshake between the infra owners
-and the application owners that allows them to be independent actors.
-Route-owners can specify that they will bind with all Gateways in the cluster,
-or only Gateways from a specific Namespace, with a specific label selector, or
-an individual Gateway. Similarly, Gateways provide the same level of control.
-This allows a cluster to be more self-governed, which requires less central
-administration to ensure that Routes are not over-exposed.
+resources they permit attaching to. It is a handshake between the infra owners
+and the application owners that allows them to be independent actors. Routes
+can only attach to specified gateways and Gateways provide a similar level of
+selection control over which Routes they will send traffic to by trusting
+specific namespaces in which those Routes live and operate. This allows a
+cluster to be more self-governed, which requires less central administration
+to ensure that Routes are not over-exposed.
```

Comment:
I think this should be updated more significantly. It all seems a bit too close to the v1alpha1 docs with the mentions of selection. In the new model, Routes directly reference Gateways they want to attach to, and Gateways describe where they trust Routes to attach from. These docs got lost in the recent restructuring, but might be a helpful reference point for rewriting this: https://deploy-preview-762--kubernetes-sigs-gateway-api.netlify.app/v1alpha2/concepts/api-overview/#how-it-works.

Reply:
890861739ef42a42882a1241b3ae005cd833c301 was an attempt to significantly reword this area of the guide based on feedback, let me know your thoughts and if you feel there's more that needs to be done here.

Reply:
I think what you have is looking pretty good, but I'd probably change this to reference that Gateways *allow* attachment, and Routes *request* attachment.

So, this section should be something like (sorry Github won't let me suggest here for some reason):
```
As a result, Gateways and Routes have independent control to determine which
resources will succeed in attaching. It is a handshake between the infra owners
and the application owners that allows them to be independent actors. Routes
can only attach to specified gateways and Gateways provide a similar level of
selection control over which Routes they allow to attach to by trusting
specific namespaces in which those Routes live and operate. This allows a
cluster to be more self-governed, which requires less central administration
to ensure that Routes are not over-exposed.
```

I also think we need to be really careful with "forward traffic to" because that implies the existence of an extra routing hop, which won't be the case with most implementations.

I think that for the attachment process, we need to make sure we're talking purely in terms of the Kubernetes objects and how they interact.

Reply:
I made the changes you requested in e5488f640e715adefe133c65a16f89e4d14a5cbd and verified that the word "forwarding" is being used in the technical sense and is not misrepresented elsewhere in the doc. Let me know if there's anything further you want to see changed.